easybuild/easyconfigs/c/crb-blast/crb-blast-0.6.9-intel-2017a-Ruby-2.3.4.eb

```diff
@@ -0,0 +1,56 @@
+easyblock = 'Bundle'
+
+name = 'crb-blast'
+version = '0.6.9'
+
+rubyver = '2.3.4'
+versionsuffix = '-Ruby-%s' % rubyver
+
+homepage = 'https://github.com/cboursnell/crb-blast'
+description = """Conditional Reciprocal Best BLAST - high confidence ortholog assignment.
+ CRB-BLAST is a novel method for finding orthologs between one set of sequences and another.
+ This is particularly useful in genome and transcriptome annotation."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+dependencies = [
+    ('Ruby', rubyver, '', True),
+    ('BLAST+', '2.6.0', '-Python-2.7.13'),
+]
+
+ext_options = {
+    'source_urls': ['http://rubygems.org/downloads/'],
+    'source_tmpl': '%(name)s-%(version)s.gem',
+}
+
+# this is a bundle of Ruby gems
+exts_defaultclass = 'RubyGem'
+
+exts_list = [
+    ('facade', '1.0.7', dict(ext_options.items() + [
+        ('checksums', ['2d33a1d839785fcb56b7da97e8aeef81b44474f927f8234c90ea18201eab597d'])])),
+    ('pathname2', '1.8.0', dict(ext_options.items() + [
+        ('checksums', ['3a315689d0f183409504bd4949d188e28559d7b8f61960c36b332481e2443472'])])),
+    ('fixwhich', '1.0.2', dict(ext_options.items() + [
+        ('checksums', ['c6a8f796a7eb60ffbc29f0d2af85461761a36c2864d25e445ff18bfbd1657078'])])),
+    ('bindeps', '1.2.1', dict(ext_options.items() + [
+        ('checksums', ['3c11d75aa722bed67246852bb430a182361a128910d384b664b91f3e65bc34b5'])])),
+    ('threach', '0.2.0', dict(ext_options.items() + [
+        ('checksums', ['432cbf3569bf9b09e26f93d0959fd6fb911c71e790e8a4cc4d1110e139a2ffca'])])),
+    ('bio', '1.5.1', dict(ext_options.items() + [
+        ('checksums', ['896c19af7e724e038baceae20c00688872b70c69ef966ef3adc42696d001b441'])])),
+    ('trollop', '2.1.2', dict(ext_options.items() + [
+        ('checksums', ['88422e8137b1e635ed07f6b8480c2c2a16d3ac1288023688c4da20d786f12510'])])),
+    (name, version, dict(ext_options.items() + [
+        ('checksums', ['69c346e7d83efe9b9a383a39b57e7cce186a82b7074f275b14906f8f05678e3e'])])),
+]
+
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in ['bioruby', 'br_biofetch.rb', 'br_bioflat.rb', 'br_biogetseq.rb', 'br_pmfetch.rb',
+                                     'crb-blast']],
+    'dirs': ['gems'],
```

Comment:
@hajgato Problem with the `sanity_check`:
```
== 2017-09-15 16:29:13,954 build_log.py:157 ERROR EasyBuild crashed with an error (at ?:124 in __init__): Sanity check failed: no (non-empty) directory of ('gems',) in /camp/apps/eb/review/software/crb-blast/0.6.9-intel-2017a-Ruby-2.3.4 (at easybuild/framework/easyblock.py:2182 in _sanity_check_step)
```

Reply:
@verdurin Without `RubyGem` EasyBlock update (https://github.com/easybuilders/easybuild-easyblocks/pull/1247), the gems are installed into the `Ruby` root directory (which we want to avoid)